docs/workflow/testing/libraries/testing-apple.md

```diff
@@ -1,34 +1,89 @@
-# Testing Libraries on iOS and tvOS
+# Testing Libraries on iOS, tvOS, and MacCatalyst
 
-In order to build libraries and tests for iOS or tvOS you need recent version of XCode installed (e.g. 11.3 or higher).
+## Prerequisites
 
-Build Libraries for iOS Simulator:
+- XCode 11.3 or higher
```

Comment:
Perhaps we can add details of needing a certificate and provisioning profile if using a device? I'm still figuring out the "barebones" setup. 

Should we also add details of downloading the different simulator OS versions (or ensuring the target OS is downloaded and perhaps provide steps such as `Xcode > Window > Devices and Simulators > + button on bottom left > OS Version dropdown selection > Download more simulator runtimes`.